AFv1/importing-data-to-afv2-mysql.md

```diff
@@ -19,7 +19,7 @@ When migration is complete your billing subscription can be canceled from the [A
 
 ### Importing Data Into CenturyLink MySQL Relational DB Service
 * To learn how to create, bind, and determine connection credentials review our Knowledge Base article on [Using CenturyLink MySQL with AppFog Applications](../AppFog/using-ctl-mysql-with-appfog-apps.md).
-* You can access your database from the command line using the credentials provided by the VCAP_SERVICES environment variable. You can also [Connect to MySQL Relational DB over SSL on AppFogv2](../Database/connecting-to-mysql-rdbs-over-ssl-on-appfog.md). The certificate needs to be in proper .pem format:
+* You can access your database from the command line or a third-party tool using the credentials provided by the VCAP_SERVICES environment variable. The VCAP_SERVICES variable can be viewed using `cf env <APP_NAME>` from the CLI, or by navigating to your application in the web control portal by selecting the AppFog icon, choosing the application, then selecting Environment. You can also [Connect to MySQL Relational DB over SSL on AppFogv2](../Database/connecting-to-mysql-rdbs-over-ssl-on-appfog.md). The certificate needs to be in proper .pem format:
```

Comment:
Change link to `/knowledge-base/database/connecting-to-mysql-rdbs-over-ssl-on-appfog`